coalib/bearlib/abstractions/Linter.py

```diff
@@ -57,6 +58,33 @@ def _prepare_options(options):
 
         options['output_regex'] = re.compile(options['output_regex'])
 
+        supported_names = {
+            'origin',
+            'message',
+            'severity',
+            'line',
+            'column',
+            'end_line',
+            'end_column',
+            'additional_info'
+        }
+        no_of_non_named_groups = (options['output_regex'].groups
+                                  - len(options['output_regex'].groupindex))
+
+        if no_of_non_named_groups:
+            logging.warning('{}: Using unnecessary capturing groups '
```

Comment:
@makman2 I'm not quite sure of this.. Don't coala has a centralised `logger` object already?